### PR TITLE
Implement `browsingContext.captureScreenshot` command

### PR DESCRIPTION
**Pull request by thiagowfx**
Bug: #514
Spec: https://w3c.github.io/webdriver-bidi/#command-browsingContext-captureScreenshot

### CODE REVIEW

File: src/bidiMapper/domains/context/browsingContextProcessor.ts [hunk 1]
```diff
@@ -237,6 +237,21 @@ export class BrowsingContextProcessor {
     );
   }
 
+  async process_browsingContext_captureScreenshot(
+    params: BrowsingContext.CaptureScreenshotParameters
+  ): Promise<BrowsingContext.CaptureScreenshotResult> {
+    const context = this.#browsingContextStorage.getKnownContext(
+      params.context
+    );
+    // TODO: return context.captureScreenshot(params);
```
Comment (sadym-chromium) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:246]: Correct. And `context.captureScreenshot` in turn should call `await this.#cdpClient.sendCommand('Page.captureScreenshot', {})`

**Comment by thiagowfx** [comment]
c.f. https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-captureScreenshot

**Comment by thiagowfx** [comment]
Capture screenshot base64:
image

**Comment by thiagowfx** [comment]
Other than improving the E2E test, this is complete.

**Comment by thiagowfx** [comment]
@sadym-chromium Test is now fully complete as we discussed it, comparing the output with a hard-coded golden string.

**Comment by thiagowfx** [comment]
The issue is that the screenshot seems to be different depending on the environment where the test is run, likely because of the viewport size. For example, on my laptop it's currently passing, but on GitHub actions it is failing.

There are no viewport params at the moment (in the spec, that is).

Should we revert to comparing it with `ANY_STR`?

**Comment by sadym-chromium** [comment]
> The issue is that the screenshot seems to be different depending on the environment where the test is run, likely because of the viewport size. For example, on my laptop it's currently passing, but on GitHub actions it is failing.
> 
> There are no viewport params at the moment (in the spec, that is).
> 
> Should we revert to comparing it with `ANY_STR`?

I'd rather get a piece of result, let's say 100x100 pixels, and compare it vs golden standard.

**Comment by sadym-chromium** [comment]
another option is to get the window size, generate expected image and compare the result to it.

**Review by sadym-chromium — APPROVED** [review]

**Comment by thiagowfx** [comment]
We need to focus the active tab with CDP Command `Page.bringToFront`, or else the screenshot action blocks in headful mode. This was discussed off-line with @sadym-chromium.

I confirm this works as we predicted.

**Comment by thiagowfx** [comment]
Added a TODO to reflect the custom tab focus quirk.

**Pull request merged**